reference/platform-app-yaml.md

```diff
@@ -390,8 +398,10 @@ relationships:
     solr: "solr:solr"
     redis: "redis:redis"
 web:
-    document_root: "/"
-    passthru: "/index.php"
+    locations:
+        "/":
+            root: "public"
+            passthru: "/index.php"
```

Comment:
OK. In that case it makes sense (I had assumed the passthru was also a path on disk)